### ISSUE TITLE
_version.py not available

### ISSUE THREAD

**Issue by johannes-moegerle** [tenpy/tenpy#274]
Following the **Quick-setup of a development environment with conda** in the documentation https://tenpy.readthedocs.io/en/latest/install/from_source.html, i.e.:
```
git clone https://github.com/tenpy/tenpy TeNPy
cd TeNPy
conda env create -f environment.yml -n tenpy_dev
conda activate tenpy_dev
pip install -e .
```
does not create a __version.py_ file, and thus when importing tenpy does not use the compiled code and throws the following warning:
Compiled, but tenpy/_version.py not available!

This is probably related to #237.

**Comment by Jakob-Unfried** [comment]
Thanks for the report!
Looks like `write_underscore_version_py` did not live up to its name...

Could you check if #275 fixes the problem also in your setup?

**Comment by jhauschild** [comment]
Actually, I solved this in 420e064e the other way around by removing the write_underscore_python alltogether. The point of having it was to notify users when they need to recompile, but I don't think writing the _version is a good solution for this anyways - it just compares the release version numbers, instead of whether the _npc_helper.pyx changed...

**Comment by johannes-moegerle** [comment]
#275 did not fix it, but also adding write_underscore_version_py() to the `if __name__ == '__main__':` would fix it.
However, if you were anyway removing it, that is also fine.

Thanks for the quick replies! :+1:
